lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
```

Comment:
I think we may want to make this change to the root django template for all in-course pages to be consistent. I'll talk with eng about that.